source/quick-start.txt

```diff
@@ -0,0 +1,136 @@
+     editor. Replace the ``<username>`` and ``<password>`` placeholders with
+     your database user's username and password.
+
+     Save this file to a safe location for use in the next step.
+
+   .. step:: Set Your Connection String
+
+      Run the following code in your shell to save the MongoDB connection string
+      in your copy buffer from the previous step to an 
+      environment variable. This method is safer than including your credentials in your source 
+      code. 
+
+   .. code-block:: bash
+
+      export MONGODB_URI='<your connection string>'
+
+   .. important::
+
+      Make sure to replace the ``<username>`` and ``<password>`` sections of the connection 
+      string with the username and password of your Atlas user.
+
+Set Up Your Project
+-------------------
+
+.. procedure::
+   :style: connected
+
+   .. step:: Create the Project
+
+      Create a new directory and initialize your project with the ``dotnet new`` command,
+      as follows:
+
+      .. code-block:: shell
+
+         mkdir entity-quickstart
+         cd entity-quickstart
+         dotnet new console
+
+   .. _entity-framework-add-mongodb-dependency:
+
+   .. step:: Add the {+provider-long+} as a Dependency
+
+      Use the ``dotnet add`` command to add the {+provider-short+} to your project as a
+      dependency.
+
+      .. code-block:: shell
+
+         dotnet add package {+package+} 
+
+Query Your MongoDB Cluster from Your Application
+------------------------------------------------
+
+.. procedure::
+   :style: connected
+
+   .. step:: Create the Code File
+
+      Open the file named ``Program.cs`` in the base directory of your project. Copy the
+      following sample code into ``Program.cs``: 
+
+      .. literalinclude:: /includes/quick-start/Program.cs
+         :language: csharp
+         :dedent:
+
+   .. step:: Run an Atlas Query
+
+      Run the following command in your shell. It should print the plot of the movie
+      "Back to the Future" from the sample dataset:
+
+      .. io-code-block::
+         :copyable: true
+
+         .. input::
+            :language: none
+            
+            dotnet run entity-quickstart.csproj
+
+         .. output::
+            :language: none
+            :visible: false
+
+            A young man is accidentally sent 30 years into the past in a time-traveling
+            DeLorean invented by his friend, Dr. Emmett Brown, and must make sure his
+            high-school-age parents unite in order to save his own existence.
+
+      .. tip::
+
+         If your output is empty, ensure you have loaded the
+         :atlas:`sample datasets </sample-data/>` into your cluster.
+
+After completing these steps, you should have a working {+framework+} application that
+connects to your MongoDB cluster, runs a query on the
+sample data, and prints out the result.
+
+Next Steps
+----------
+
+Learn how to use the {+provider-short+} to perform common operations in Quick Reference.
```

Comment:
Suggestion:

I am not sure whether this was intentionally left out, but I think this is missing a link.